velox/type/DecimalUtil.h

```diff
@@ -203,32 +204,57 @@ class DecimalUtil {
     return static_cast<TOutput>(rescaledValue);
   }
 
-  /// Rescales a double value to decimal value of given precision and scale. The
-  /// output is rescaled value of int128_t or int64_t type. Returns error status
-  /// if fails.
-  template <typename TOutput>
-  inline static Status
-  rescaleDouble(double value, int precision, int scale, TOutput& output) {
+  /// Rescales a floating point value to decimal value of given precision and
```

Comment:
perhaps, clarify that TInput can be only float or double

```
@tparam TInput Either float or double.
@tparam TOutput Either int64_t or int128_t.
```

Reply:
Thanks, updated.